File: src/main/java/org/manapart/jumpPads/ModItems.java
package org.manapart.jumpPads;

import net.minecraft.item.Item;
import net.minecraftforge.event.RegistryEvent;
import net.minecraftforge.eventbus.api.SubscribeEvent;
import net.minecraftforge.fml.common.Mod;
import net.minecraftforge.registries.ForgeRegistries;

import static org.manapart.jumpPads.JumpPads.*;

@Mod.EventBusSubscriber(bus = Mod.EventBusSubscriber.Bus.MOD)
public class ModItems {

    @SubscribeEvent
    public static void registerItems(RegistryEvent.Register<Item> event) {
        if (!ForgeRegistries.ITEMS.containsKey(JUMP_PAD_ITEM.getRegistryName())) {
//            System.out.println("Registering items");
            ForgeRegistries.ITEMS.register(jumpPadsIcon);
            ForgeRegistries.ITEMS.register(JUMP_PAD_ITEM);
            ForgeRegistries.ITEMS.register(JUMP_PAD_ITEM_2);
            ForgeRegistries.ITEMS.register(JUMP_PAD_ITEM_3);
        }
    }

}

File: src/main/java/org/manapart/jumpPads/ModItems.kt
package org.manapart.jumpPads

import net.minecraftforge.registries.DeferredRegister
import net.minecraftforge.registries.ForgeRegistries
import thedarkcolour.kotlinforforge.forge.KDeferredRegister
import thedarkcolour.kotlinforforge.forge.registerObject

object ModItems {
    val REGISTRY = DeferredRegister.create(ForgeRegistries.ITEMS, MODID)

    val JUMP_PAD_ITEM by REGISTRY.registerObject("jumppaditem") { JumpPadItem(ModBlocks.JUMP_PAD) }
    val JUMP_PAD_ITEM_2 by REGISTRY.registerObject("jumppaditem_2") { JumpPadItem(ModBlocks.JUMP_PAD_2) }
    val JUMP_PAD_ITEM_3 by REGISTRY.registerObject("jumppaditem_3") { JumpPadItem(ModBlocks.JUMP_PAD_3) }
}